liqd_product/assets/scss/_form.scss

```diff
@@ -4,9 +4,9 @@ textarea {
     box-shadow: 0 0 2px $brand-primary-tint inset;
 
     &:invalid {
-        border-color: $brand-danger;
-        box-shadow: 0 0 2px $brand-danger-tint inset;
-        color: $brand-danger;
+        border-color: $brand-primary;
+        box-shadow: 0 0 2px $brand-primary-tint inset;
+        color: $brand-primary;
```

Comment:
last note: in the base _form.scss the color is set to `contrast-color($body-bg);`  [due to the input property `background-color: $body-bg;`]

for uniformity we should use that, too